src/include/txn.i

```diff
@@ -10,58 +10,111 @@ static inline int __wt_txn_id_check(WT_SESSION_IMPL *session);
 static inline void __wt_txn_read_last(WT_SESSION_IMPL *session);
 
 #ifdef HAVE_TIMESTAMPS
+#if WT_TIMESTAMP_SIZE == 8
```

Comment:
I am building with new size to test it out. But then I have only tested the python test suite as of now.